collection/vivek.json

```diff
@@ -0,0 +1,10 @@
+{
+  "name": "Vivek",
```

Comment:
Can you please update the `name` field to use lowercase letters. This should match the JSON file name.